docs/language-server/protocol-language-server.md

```diff
@@ -376,6 +376,11 @@ interface ExpressionUpdate {
    *  - array with multiple values represents an intersetion type
    */
   type: string[];
+  /**
+   * The list of types this expression can be converted to.
+   */
+  hiddenType: string[];
```

Comment:
Older version of the protocol might not support this attribute. Shouldn't it thus be optional?

Reply:
Gui allows messages to have extra fields. It only checks ones that it is interested in.